File: 1 to 100/C++/58_length-of-last-word.cpp
#include<bits/stdc++.h>
using namespace std;
#define endl '\n'
#define int long long
const int mod = 1e9+7;

class Solution {
public:
    int lengthOfLastWord(string s) {
        int i = s.size()-1;
        while(i>=0 && s[i] == ' '){
            i--;
        }
        int j = 0;
        while(i>=0 && s[i] != ' '){
            i--;
            j++;
        }
        return j;
    }
};

signed main(){
    ios_base::sync_with_stdio(false);
    cin.tie(0);cout.tie(0);
    Solution obj;
}

File: 1 to 100/Python/58_length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        out = [char for char in s][::-1]
        while out[0] == " ":
            out.remove(" ")
        if len(out) <2:
            return len(out)
        for i in range(len(out)):
            if out[i] == " ":
                break
            if i == len(out) -1:
                i += 1
                break
        return i